tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -112,7 +112,8 @@ def tap_comment(self):
         element.tap()
         self.marionette.execute_script(
             'arguments[0].scrollIntoView(true);', [element])
-        element = self.marionette.find_element(*self._comment_locator)
+        comment = self.marionette.find_element(*self._comment_locator)
+        Wait(self.marionette, timeout=120).until(expected.element_displayed(comment))
         self.marionette.execute_script(
             'arguments[0].scrollIntoView(true);', [element])
         element.tap()
```

Comment:
wait, if you changed the above variable to comment, then you should also change line 118 and 119 to use the 'comment' as well and not 'element.  This would surely cause some unintended behavior.